backend/lib/auth.controller.js

```diff
@@ -1,7 +1,9 @@
 exports.google = (req, res) => {
     const io = req.app.get('io')
+    console.log(req.user)
```

Comment:
Will we need it? Otherwise, can you remove the log?

Reply:
I will remove it